packages/types/src/files.ts

```diff
@@ -25,21 +25,23 @@ export type GetPresignedURLForFileArguments = {
 
 export type GetPresignedURLForFileResponse = {
   data: {
-    url: string;
+    url: getPresignedURLForFile;
   };
 };
 
+type getPresignedURLForFile = string;
+
 export type ListFilesArguments = {
   organizationID?: string;
   cursor?: string;
 };
 
-export type ListFilesResponse = {
+export type ListFilesResponse = ListFilesMetadata & {
   data: FileRecord[];
-  hasMore: boolean;
-  cursor?: string;
 };
 
+type ListFilesMetadata = PaginationMetadata;
```

Comment:
listFiles method will return the list of files directly and any pagination data inside the metadata, like so:
```ts
const [fileRecords, { metadata } ] = await slashauthClient.listFiles(org);

if (metadata.hasMore) slashauthClient.listFiles(org, metadata.cursor)
```